sql/mob_pools.sql

```diff
@@ -3124,7 +3124,7 @@ INSERT INTO `mob_pools` VALUES (3068,'Osschaarts_Wyvern','Osschaarts_Wyvern',193
 INSERT INTO `mob_pools` VALUES (3069,'Oupire','Oupire',284,0x0000350700000000000000000000000000000000,4,5,7,220,100,0,1,1,0,2,7,0,0,1157,8,0,150,0,0,284,284);
 INSERT INTO `mob_pools` VALUES (3070,'Ouryu','Ouryu',259,0x0000620200000000000000000000000000000000,7,4,7,240,125,1040,1,1,0,2,0,32,297,1179,0,0,109,0,0,259,259);
 INSERT INTO `mob_pools` VALUES (3071,'Ouzelum','Ouzelum',125,0x0000530100000000000000000000000000000000,4,5,12,240,100,0,0,0,0,2,0,0,1,1179,0,0,266,0,0,125,125);
-INSERT INTO `mob_pools` VALUES (3072,'Overgrown_Ivy','Overgrown_Ivy',186,0x00007C0100000000000000000000000000000000,2,2,7,360,100,0,1,0,0,0,0,0,1261,1155,8,0,0,0,0,186,186);
+INSERT INTO `mob_pools` VALUES (3072,'Overgrown_Ivy','Overgrown_Ivy',186,0x00007C0100000000000000000000000000000000,2,2,7,100,100,0,1,0,0,0,0,0,1261,1155,8,0,0,0,0,186,186);
```

Comment:
Do we have a source for that 100 delay? Wiki old wiki says fast and compares to hundred fists, and that's mostly likely bull. They've been wrong just about everytime they say that (usually some kind of multihit involved). Bg doesn't mention it at all. JP Wiki also has nothing. I see it is listed as mnk mob on old wiki and a warrior on bg tho. 

I have a hunch it just has a high rate of double attacks as war/war or is war/mnk or mnk/war.

Reply:
Err on side of caution of you want to get the pr in and do further adjustments later.

Reply:
ps. I **did** watch the video. it didn't look like the swings were constant to me. I can't measur eby eyeballing it.

Reply:
I honestly don't have a straight answer for you other than watching like 4-5 videos and trying to get it to  match the best I can. 100 Delay seems to fit with the constant auto attacks with a 10% double attack proc. Any higher percentage, even with a slower delay just didn't feel right. I also found all the same thing you did so I can't say for certain this is 100% accurate but it's the closest I can get it. If someone else to play around with it I will gladly change it.

The inconsistent swings must be a double attack percentage bc in nowhere of the few videos ive watched did I ever see a triple attack. In the cases it may have looked like a triple attack I had a few other ppl also look at it and they agreed it seemed to be the animation that happens when it doubles+autos right after the 2nd hit, which also led be to believe it must be a constant low delay.

Reply:
And it does not look like a multi-hit either otherwise this thing would be going completely crazy

Reply:
There were a few pauses when I watched that made me think it might be just a slow conistant2-hit occasionally being 3, or it might have just been a bit of frame stuffer with non stop single swings. Def not a stream of high speed triple attack like I've seen with most mob wiki says "fast like hundred fists".

Didn't really look fast enough to be 100 to my own eyes, maybe 150. But Wouldn't be able to say without some kind of reliable measurement. I think we want to err on the side of making the mob harder than should be rather than being to easy so its probably going to merge as is unless I get some better data but wanted to check in case you did something to figure out it was 100 or had some reliable source for the number. 

Winter was playing around some math to see if there was a way we could figure it out better but we couldn't get anything concrete without popping it ourselves yet.

Reply:
Can you tag a todo comment here for me? I'll help me revisit in the future
`-- Todo: verify melee delay`

Reply:
> Winter was playing around some math to see if there was a way we could figure it out better but we couldn't get anything concrete without popping it ourselves yet.

Yeah I would do the capture myself but I did CoP on my retail char a long time ago. Sorry :( had to rely on what information I had.

Reply:
Its ok man, I just always gotta check where the numbers are obtained so I know, ya know? :)

Reply:
Of course :D

Reply:
by todo comment, I meant here in file as `-- Todo: verify melee delay` at the end of the line.